docs/using.rst

```diff
@@ -611,6 +618,30 @@ The following commands could be used to specify where these files are located::
   sed -i 's,/etc/letsencrypt/live/example.com,/home/user/me/certbot,g' /etc/letsencrypt/renewal/example.com.conf
   certbot update_symlinks
 
+Automated Renewals
+------------------
+
+Many Linux distributions provide automated renewal when you use the
+packages installed through their system package manager.  The
+following table is an *incomplete* list of distributions which do so,
+as well as their methods for doing so.
+
+If you are not sure whether or not your system has this already
+automated, refer to your distribution's documentation, or check your
+system's crontab and systemd timers.
```

Comment:
We should provide instructions on how to do this, or link to them.

Reply:
Good point - maybe Harlan can take a stab at that? I can't seem to even find the debian documentation

Reply:
Well... to check systemd timers, you'd do `systemctl ilst-timers`.  For cron, you'd check `/etc/crontab` and `/etc/cron.*`.  And, I guess, `/etc/anacrontab`.  That's the standard locations, at least.  But distributions could customize it; it's true for Debian/Ubuntu/CentOS/RHEL, but, for all *nix?

Reply:
I think providing the typical examples here is good enough.